repos/system_upgrade/el9toel10/actors/sssd/sssdchecks/libraries/sssdchecks.py

```diff
@@ -0,0 +1,30 @@
+from leapp import reporting
+
+
+def check_config(model):
+    if not model:
+        return
+
+    # If sss_ssh_knownhostsproxy was not configured, there is nothing to do
+    if not model.ssh_config_files:
+        return
+
+    summary = (
+        'SSSD\'s sss_ssh_knownhostsproxy tool is replaced by the more '
+        'reliable sss_ssh_knownhosts tool. SSH\'s configuration will be updated '
+        'to reflect this by updating every mention of sss_ssh_knownhostsproxy '
+        'by the corresponding mention of sss_ssh_knownhosts.\n'
+        'SSSD\'s ssh service will be enabled if not already done.'
```

Comment:
DO NOT DO THIS yet. For now I will use this comment as draft after we discuss the enablement of the SSSD ssh service, we can sync off GitHub.
```suggestion
    'The sss_ssh_knownhostsproxy tool, which is part of SSSD, was replaced by'
    ' sss_ssh_knownhosts tool in RHEL 10. The SSSD and SSH configuration will be'
    ' updated to use sss_ssh_knownhosts by replacing every occurrence of'
    ' sss_ssh_knownhostsproxy with sss_ssh_knownhosts, comments included. The'
    ' following files will be updated:{}{}'.format(FMT_LIST_SEPARATOR, model.ssh_config_files)
```

The FMT_LIST_SEPARATOR is a "standard" separator we use for lists, you can just put this as global variable.
```python
FMT_LIST_SEPARATOR = "\n    - "

```

Also the severity of the report might change if we are gonna enable something that was previously disabled.

Reply:
I applied this change, but only the formatting part. There is still a single report as agreed out on Slack.

Reply:
I don't think the report text is updated. And also please use the FMT_LIST_SEPARATOR.
My suggestion is missing the line about enabling SSSD's ssh service, please include that before the "The following files will be updated" line

Reply:
Done your way.

I find it a strange way of reporting the files, but if that's the Leapp way, let's do it that way.